lib/features/seller/authentication/model/seller_register_request_model.dart

```diff
@@ -0,0 +1,72 @@
+class SellerRegisterRequestModel {
+  SellerRegisterRequestModel({
+    required this.firstName,
+    required this.lastName,
+    this.displayName,
```

Comment:
In the payload we don't have these keys displayName and idToken for now. So the API is failing

<img width="1129" alt="Screenshot 2024-11-19 at 9 06 12 PM" src="https://github.com/user-attachments/assets/e614496f-c2bb-4da9-9852-a2cf9d1a1676">

Reply:
Now i have changed this to phone number registration. So it will work fine now.